### DISCUSSION TITLE
b-form-radio-group should not trigger input event when value is changed from outside

### DISCUSSION THREAD

**Discussion by upsilon** [bootstrap-vue/bootstrap-vue#5908] [Q&A] [answered]
### Describe the bug

The input event is triggered even if the value is changed from outside the `<b-form-radio-group>` component.
This behavior differs from the native `<input type="radio">` element.

### Steps to reproduce the bug

https://jsfiddle.net/kim_upsilon/u7vcz89o/

![](https://user-images.githubusercontent.com/92577/96197149-2e351180-0f8c-11eb-8d34-19b3a38bce05.png)

When the selectedValue is changed by clicking native radio element, the value of b-form-radio-group is also changed accordingly.
At this time, the input event is triggered from **both** b-form-radio-group and native radio element.

In the case of the native input element, the input event does not trigger if the value is changed from outside the element.
https://html.spec.whatwg.org/multipage/input.html#common-input-element-events

### Expected behavior

On the above JSFiddle link, clicking either bvRadio or nativeRadio will occur the input event only once.

### Versions

**Libraries:**

- BootstrapVue: 2.17.3
- Bootstrap: 4.5.2
- Vue: 2.6.12

**Environment:**

- OS: Windows 10 Version 2004
- Browser: Google Chrome
- Version: 86.0.4240.75

**Comment by Hiws** [comment, marked as answer]
This is to be expected since Vue relies on the `input` event when binding `v-model` on custom components.
If you only want the event to fire when it was changed due to user interaction, you should use the `change` event instead.
